gz_ros2_control/src/gz_system.cpp

```diff
@@ -336,7 +355,6 @@ bool IgnitionSystem::initSim(
           this->dataPtr->joints_[j].joint_velocity_cmd = initial_velocity;
         }
       } else if (joint_info.command_interfaces[i].name == "effort") {
-        this->dataPtr->joints_[j].joint_control_method |= EFFORT;
```

Comment:
Was this removed intentionally?

Reply:
yes, it's intentionally

Reply:
Why was this removed? I believe it makes sense, but would like to understand.

Reply:
because this is not part of the registration